chia/policy/fee_estimator_demo.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+
+from chia.full_node.fee_estimate import FeeEstimate
+from chia.util.ints import uint64
+
+MIN_MOJO_PER_COST = 5
+
+
+def demo_fee_rate_function(cost: int, time_in_seconds: int) -> uint64:
+    return uint64(cost * MIN_MOJO_PER_COST * max((3600 - time_in_seconds), 1))
```

Comment:
I see. This is a place holder for the actual fee-estimator logic. Right?

Reply:
That's right.